src/Console/Command/DescribeCommand.php

```diff
@@ -156,6 +156,10 @@ private function describeRule(OutputInterface $output, $name)
             $description .= sprintf(' <error>DEPRECATED</error>: %s.', $message);
         }
 
+        if ($output->getVerbosity() >= OutputInterface::VERBOSITY_VERBOSE) {
+            $output->writeln(sprintf('Fixer class: <comment>%s</comment>.', get_class($fixer)));
+        }
+
         $output->writeln(sprintf('<info>Description of</info> %s <info>rule</info>.', $name));
```

Comment:
I would put class name after `rule` , as user was asking for `rule`, so let us display `rule` first, always (regardless verbosity)